tests/utils.go

```diff
@@ -654,9 +651,6 @@ func cleanNamespaces() {
 		// Remove all Pods
 		PanicOnError(virtCli.CoreV1().RESTClient().Delete().Namespace(namespace).Resource("pods").Do().Error())
 
-		// Remove all VM Secrets
-		PanicOnError(virtCli.CoreV1().RESTClient().Delete().Namespace(namespace).Resource("secrets").Do().Error())
```

Comment:
I think that we  should keep that, but limit it to secrets which are marked with `kubevirt.io`.